packages/contracts-bedrock/scripts/checks/common/artifact.go

```diff
@@ -0,0 +1,223 @@
+package common
+
+type ArtifactFile struct {
+	Abi               []AbiEntry        `json:"abi"`
+	Bytecode          BytecodeObject    `json:"bytecode"`
+	DeployedBytecode  BytecodeObject    `json:"deployedBytecode"`
+	MethodIdentifiers map[string]string `json:"methodIdentifiers"`
+	RawMetadata       string            `json:"rawMetadata"`
+	Metadata          MetadataObject    `json:"metadata"`
+	StorageLayout     *StorageLayout    `json:"storageLayout,omitempty"`
+	Ast               AST               `json:"ast"`
+	Id                int               `json:"id"`
+}
+
+type BytecodeObject struct {
+	Object         string                 `json:"object"`
+	SourceMap      string                 `json:"sourceMap"`
+	LinkReferences map[string]interface{} `json:"linkReferences"`
+}
+
+type MetadataObject struct {
+	Compiler CompilerInfo          `json:"compiler"`
+	Language string                `json:"language"`
```

Comment:
We have a `solc` package that would make sense to include these types, see https://github.com/ethereum-optimism/optimism/blob/develop/op-chain-ops/solc/types.go

Reply:
Moved everything to the Solc package